_posts/2016-10-07-under-the-decision-tree-4.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "Under the Decision Tree (#4)"
+date: 2016-10-07 07:00:00
+tags: 
+- Machine Learning
+categories: Under the Decision Tree
+twitter_text: Under the Decision Tree (#4) #machinelearning #AI #artificialintelligence #software #bigdata #science
+authors: Scott Schwalm
+image: /images/tree-338211_1280.jpg
+external_links_target_blank: true
+---
+
+Welcome back for another edition of **Under the Decision Tree**.  This week we had The Data Science Conference in Seattle and interesting articles from teaching AI to be sarcastic to predictions of what AI will look like in 2030.  
```

Comment:
> interesting articles from teaching AI to be sarcastic to predictions of what AI will look like in 2030.  

this could read better.

> interesting articles that include teaching AI to be sarcastic, predictions of what AI will look like in 2030, and much more.